.NET/Microsoft.Recognizers.DataTypes.DateTime/Microsoft.Recognizers.DataTypes.DateTime.csproj

```diff
@@ -1,8 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
```

Comment:
To match the other modules in the project, let's rename Microsoft.Recognizers.DataTypes.DateTime to Microsoft.Recognizers.Text.DataTypes.TimexExpression (project/assembly and namespace)
Can you also rename the javascript project accordingly?

Reply:
The javascript module could be @microsoft/recognizers-text-datatypes-timex-expression to match the others.
What do you think?